tsweb/tsweb_test.go

```diff
@@ -925,8 +925,9 @@ func TestStdHandler_ConnectionClosedDuringBody(t *testing.T) {
 		t.Fatalf("making request: %s", err)
 	}
 	res, err := http.DefaultClient.Do(req)
+	close(requestCanceled)
 	if !errors.Is(err, context.Canceled) {
-		t.Errorf("got error %v, want context.Canceled", err)
+		t.Errorf("got error %v, want nil or context.Canceled", err)
```

Comment:
The error message says that `nil` is ok, but the condition is still `errors.Is(err, context.Canceled)`